source/installation/deployment-and-management.rst

```diff
@@ -0,0 +1,452 @@
+
+   * - :envvar:`MINIO_ROOT_PASSWORD`
+     - The corresponding secret key to use for the 
+       :ref:`root <minio-users-root>` user.
+
+       Replace this value with a unique, random, and long string.
+
+   * - :envvar:`MINIO_KMS_SECRET_KEY`
+     - The key to use for encrypting the MinIO backend (users, groups,
+       policies, and server configuration).
+
+       Use the following format when specifying the encryption key:
+
+       ``<key-name>:<encryption-key>``
+
+       - Replace the ``<key-name>`` with any string. You must use this
+         key name if you later migrate to using a dedicated KMS for 
+         managing encryption keys.
+
+       - Replace ``<encryption-key>`` with a 32-bit base64 encoded value.
+         For example:
+
+         .. code-block:: shell
+            :class: copyable
+   
+            cat /dev/urandom | head -c 32 | base64 -
+
+         Save the encryption key to a secure location. You cannot restart the
+         MinIO server without this key.
+
+   * - ``/data``
+     - The path to each disk on the host machine. 
+
+       See :mc-cmd:`minio server DIRECTORIES` for more information on
+       configuring the backing storage for the :mc:`minio server` process.
+
+       MinIO writes objects to the specified directory as is and without
+       :ref:`minio-erasure-coding`. Any other application accessing that
+       directory can read and modify stored objects.
+
+   * - ``--console-address ":9001"``
+     - The static port on which the embedded MinIO Console listens for incoming
+       connections.
+
+       Omit to allow MinIO to select a dynamic port for the MinIO Console. 
+       With dynamic port selection, browsers opening the root node hostname 
+       ``https://minio1.example.com:9000`` are automatically redirected to the
+       Console.
+
+You may specify other :ref:`environment variables 
+<minio-server-environment-variables>` as required by your deployment.
+
+4) Open the MinIO Console
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open your browser to the DNS name or IP address corresponding to the 
+container and the MinIO Console port. For example,
+``https://127.0.0.1:9001``.
+
+Log in with the :guilabel:`MINIO_ROOT_USER` and :guilabel:`MINIO_ROOT_PASSWORD`
+from the previous step.
+
+.. image:: /images/minio-console-dashboard.png
+   :width: 600px
+   :alt: MinIO Console Dashboard displaying Monitoring Data
+   :align: center
+
+You can use the MinIO Console for general administration tasks like
+Identity and Access Management, Metrics and Log Monitoring, or 
+Server Configuration. Each MinIO server includes its own embedded MinIO
+Console.
+
+Applications should use the ``https://HOST-ADDRESS:9000`` to perform S3
+operations against the MinIO server.
+
+.. _deploy-minio-standalone-container:
+
+Deploy Standalone MinIO in a Container
+--------------------------------------
+
+The following procedure deploys a single MinIO container in with single drive or
```

Comment:
`The following procedure deploys a single MinIO container in with single drive` -> `The following procedure deploys a single MinIO container with single drive`